triggers/message_posted_event.ts

```diff
@@ -32,9 +32,6 @@ const trigger: Trigger<typeof ReplyToMessageWorkflow.definition> = {
   name: "Reply to the posted message",
   workflow: `#/workflows/${ReplyToMessageWorkflow.definition.callback_id}`,
   inputs: {
-    // TODO: message_posted event triggers currently do not support providing a message_context data object.
-    // however, in the future it will. when that support lands, we should change these properties
-    // to be a single message_context property and type, provided by {{data.message_context}}.
```

Comment:
I looked into using the `message_context` but I couldn't get it to work. So, I've removed the `TODO` comment because we don't want our samples to have suggested future improvements.

The documentation examples and other https://github.com/slack-samples do not use `message_context` in the way that the comment described. I decided to remove the TODO entirely since other samples would also need the same update. For now, I found the `message_ts` and `channel_id` to be straight-forward and aligned with the documentation that I read.

Happy to update this if there's a reference that explains how it should be done!